addons/mass_mailing_sms/report/mailing_trace_report_views.xml

```diff
@@ -1,5 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="mailing_sms_trace_report_view_pivot" model="ir.ui.view">
```

Comment:
Just some guidelining: mailing_trace_report_sms_view_pivot (model - suffix (sms) - type (view) - subtype (pivot) . Same for name to update accordingly.